fieldservice_route/security/ir.model.access.csv

```diff
@@ -1,7 +1,10 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_fsm_route_user,fsm.route.user,model_fsm_route,fieldservice.group_fsm_user,1,0,0,0
 access_fsm_route_dispatcher,fsm.route.dispatcher,model_fsm_route,fieldservice.group_fsm_dispatcher,1,1,1,1
+access_fsm_route_portal,access.fsm.route.portal,model_fsm_route,base.group_portal,1,0,0,0
 access_fsm_route_day_user,fsm.route.day.user,model_fsm_route_day,fieldservice.group_fsm_user,1,0,0,0
 access_fsm_route_day_dispatcher,fsm.route.day.dispatcher,model_fsm_route_day,fieldservice.group_fsm_manager,1,1,0,0
+access_fsm_route_day_portal,access.fsm.route.day.portal,model_fsm_route_day,base.group_portal,1,0,0,0
 access_fsm_route_dayroute_user,fsm.route.dayroute.user,model_fsm_route_dayroute,fieldservice.group_fsm_user,1,1,0,0
 access_fsm_route_dayroute_dispatcher,fsm.route.dayroute.dispatcher,model_fsm_route_dayroute,fieldservice.group_fsm_dispatcher,1,1,1,1
+access_fsm_route_dayroute_portal,access.fsm.route.dayroute.portal,model_fsm_route_dayroute,base.group_portal,1,1,1,0
```

Comment:
Any specific reason a portal user needs to write or create a dayroute?

Reply:
@brian10048  I got a permission error in the mobile app for the portal group.

Reply:
@brian10048 Because when the portal user updates the order from the mobile side record will create or update in fsm.route.dayroute

https://github.com/OCA/field-service/blob/12.0/fieldservice_route/models/fsm_order.py#L77

Also gave ir_sequence read access to the portal user because of https://github.com/OCA/field-service/blob/12.0/fieldservice_route/models/fsm_route_dayroute.py#L84

Cc @smangukiya